draft-linker-diem-requirements.md

```diff
@@ -51,15 +51,68 @@ TODO Introduction
 
 # Use Cases and Requirements for Initial Scope
 
-## Use Cases
+# Other Use Cases and Requirements
 
-## Requirements
+# Requirements
 
-# Other Use Cases and Requirements
+## Authenticity
+
+Validators MUST be able to verify that an emblem was issued for the respective bearer, issued by the claimed issuer (where applicable), and that none of its associated data was changed.
+
+## Authorization
+
+Validators MUST be able to verify that an emblem issuer was authorized to issue the emblem.
+Authorizing parties MAY limit what emblems an issuer can issue, e.g., they MAY limit issuance to certain bearers.
+
+### Decentralized Authorization
+
+Anyone MUST be able to authorize an issuer.
+
+## Accountability
+
+Emblem issuance and authorization of issuers (where applicable) MUST be attributable to issuers and authorizing parties.
+Authorizing parties and emblem issuers MUST NOT be able to repudiate that they issued an emblem or authorization.
+
+## Revocation
+
+Emblems and authorizations (where applicable) MUST be revokable within reasonable windows of time.
+Depending on the design, it may suffice to rely on expiration of short-lived emblems and authorization effectively as a revocation mechanism.
+
+## Undetectable Validation
+
+Emblem issuers MUST NOT be able to detect whether someone is requesting or validating emblems issued by them.
+
+## Visual
+
+An emblem MUST be capable of carrying a visual representation of the physical emblem it represents.
+
+## Law Carrying
+
+An emblem MUST carry an unambiguous indication of the international law or laws conferring the semantics of the emblem.
+
```

Comment:
```suggestion
```

I suggest removing these requirements. For certain use cases, such functionality could be useful, but I think it would be better integrated in validator clients and need not be realized by the IETF standard.